components/02-elements/tab/tab.js

```diff
@@ -7,8 +7,8 @@
         mediaQuery         = window.matchMedia('(min-width: 768px)');
 
   tab.forEach(element => {
-    const title   = element.querySelectorAll('.tab__title'),
-          content = element.querySelectorAll('.tab__content');
+    const title   = element.querySelectorAll(':scope > .tab__title'),
+          content = element.querySelectorAll(':scope > .tab__content');
```

Comment:
`:scope` doesn't have enough support in browsers
https://developer.mozilla.org/en-US/docs/Web/CSS/:scope

I'd suggest to use something like this
```js
element.children.forEach(child => {
  if (child.classList.contains('tab__title')) {
    // title logic
  }

  if (child.classList.contains('tab__content')) {
    // content logic
  }
})
```